model_transaction_config.go

```diff
@@ -167,10 +169,9 @@ func (t *TransactionOutput) processOutput(ctx context.Context, cacheStore caches
 		}
 		return t.processAddressOutput()
 	} else if t.OpReturn != nil { // OP_RETURN output
-		if checkSatoshis && t.Satoshis > 0 {
```

Comment:
@mrz1836 I removed the check for satoshis on OP_RETURN outputs. I think these should be able to be 0 value outputs. Right?